docusaurus-search-local/src/client/theme/SearchBar/SearchBar.tsx

```diff
@@ -82,7 +83,7 @@ export default function SearchBar({
   } catch (e: unknown) {
     if (indexDocs) {
       if (e instanceof ReactContextError) {
-        console.error("useDocsPreferredVersion", e);
+        debug("SearchBar")("useDocsPreferredVersion", e);
```

Comment:
I think we'd better to just remove the error log. There's no need to introduce a new library on client side

Reply:
Fair enough, I've removed the error logs entirely. :+1:

Reply:
Fair enough, I've removed the error log 👍🏻 (sorry about the double comment, I tried replying via VS Code's GitHub extension and it ended up creating a new thread instead of replying to your message 😕 ).